package.json

```diff
@@ -84,13 +89,13 @@
     "istanbul": "~0.4.5",
     "mocha": "~7.1.2",
     "pkg-versions": "~2.1.0",
-    "prettier": "^2.0.5",
+    "prettier": "~2.0.5",
     "puppeteer": "~3.0.4",
     "superagent": "~5.2.2",
     "supertest": "~4.0.2"
   },
   "engines": {
-    "node": ">=10.18"
+    "node": "^10.18.0 || >=11.14.0"
```

Comment:
Something I forgot to ask: why is this needed?

Reply:
`fs.promises` was first added in `11.14.0` and later backported to `10.17.0`.
We are fine to use it if we support latest Node 10 and latest Node 11.
`>= 10.18` would technically include `11.0.0` which does not support `fs.promises` yet.

See [`eslint-plugin-node` rule](https://github.com/mysticatea/eslint-plugin-node/blob/4a348e7d24e0d806d7873f293c91a922d0316d14/lib/rules/no-unsupported-features/node-builtins.js#L136-L141).